test/utilities/oas-operation/oas-operation.test.ts

```diff
@@ -164,4 +167,20 @@ describe('OASOperation', () => {
       expect(operation.getResponseSchema(683)).toBeNull();
     });
   });
+
+  describe('getSecurity', () => {
+    it('uses operation level security if it exists', () => {
+      const secureOperation = new OASOperation({ ...baseOperation }, [
+        { 'boromir-security': [] },
+      ]);
+
+      expect(secureOperation.security).toEqual([
+        { key: 'boromir-security', scopes: undefined },
+      ]);
+    });
+
+    it('overrides spec level security with operation level security if it exists', () => {
+      expect(operation.security).toEqual([]);
+    });
```

Comment:
Not sure I understand this test

Reply:
I think I changed the test and forgot to update the description, I'll check it out